1209/ADDA/index.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: pid
+title: MicroPython Boards
+owner: micropython
```

Comment:
The capitalisation of this needs to match the org folder name for the other file, i.e. MicroPython (or rename the folder).